source/overview/mattermost-cloud-overview.rst

```diff
@@ -2,81 +2,64 @@
+To get started with Mattermost Cloud Enterprise, visit: https://mattermost.com/cloud to speak to a Mattermost Account Executive.
 
-Viewing and Managing Billing Details
-------------------------------------
+FAQ
+---
 
-You can view your billing details in **System Console > Billing & Account**.
+How much does Mattermost Cloud Cost?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Subscription
-^^^^^^^^^^^^
+Mattermost Cloud Professional is free for up to 10 users. After ten users, it is $10.00/user/month as a pay-as-you-go subscription. Please contact us for over 1000 users.
 
-The subscription screen outlines the details of your Mattermost Cloud plan, including the cost per user per month and the number of users currently registered in your workspace. You can also find a summary of your last invoice in the same page.
+Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.    
 
-Billing History
-^^^^^^^^^^^^^^^
+How can customers trial Mattermost Cloud?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Billing History screen contains a list of all your monthly invoices and payments. Invoices appear immediately after the billing period ends.
+We offer ten users free as a trial option in our Mattermost Cloud Professional edition.  Customers under 11 users can use Mattermost Cloud Professional free for as long as they need.  All features are provided in this free offering. 
 
-Billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month, at which time subscription costs are calculated and a credit card charge is automatically attempted. It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
+Mattermost Cloud Enterprise offers a discounted proof-of-concept for 60 days. The proof-of-concept program includes a dedicated customer engineer.  
 
-If you have a payment failure, your invoice will show *Payment Failed*. Please review the accuracy of your credit card information in **Payment Information**. Unresolved failed payments may result in a delinquency and an interruption to your subscription.
+When will we support other regions?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Company information
-^^^^^^^^^^^^^^^^^^^^
+Mattermost Cloud Enterprise will support data residency in early 2021. We appreciate feedback from our customers on regional support in both editions.  
 
-The **Company Information** screen contains an area to add your company address where you are physically located. The company address will appear on your invoice if specified. Otherwise the billing address associated with the credit card on file will be used.
+How is Mattermost Cloud secured?
```

Comment:
We refer to Mattermost Cloud, but also Mattermost Cloud Enterprise. We don't reference Mattermost Cloud Professional though. Is Mattermost Cloud aka Mattermost Cloud Professional?

Reply:
Mattermost Cloud incorporates both. I specified each product specifically when the information was different. Open to suggestions or feedback on being more explicit on this.